.neuro/live.yml

```diff
@@ -3,14 +3,14 @@ title: demo-oss-dogs
 id: mlops_demo_oss_dogs
 
 defaults:
-  preset: cpu-medium-p
+  preset: cpu-medium
   life_span: 1d
   env:
     # Those env vars will be assigned to every job in this flow
-    PACHY_URI: pachd.pachyderm.svc.cluster.local:30650
+    PACHY_URI: api.pachyderm.mlops.neu.ro:30650
```

Comment:
we have an updated pachy there already?
could I perform iteration over the project?

Reply:
Pachy is updated and it should work with this branch

Reply:
will do test with SA